modules/installation-vsphere-infrastructure.adoc

```diff
@@ -41,10 +41,6 @@ You must ensure that the time on your ESXi hosts is synchronized before you inst
 |Optional: Networking (NSX-T)
 |vSphere 7.0 Update 2 or later, or VMware Cloud Foundation 4.3 or later; vSphere 8.0 Update 1 or later, or VMware Cloud Foundation 5.0 or later
 |For more information about the compatibility of NSX and {product-title}, see the Release Notes section of VMware's link:https://docs.vmware.com/en/VMware-NSX-Container-Plugin/index.html[NSX container plugin documentation].
-
-|CPU micro-architecture
```

Comment:
Do we really need to remove this part? Because this doc is for fresh installation, I think this section can guide customers to choose a proper CPU architecture. WDYT?

Reply:
hi @JianLi-RH , according to @wking in https://issues.redhat.com/browse/OCPBUGS-44063 this information is only relevant for the 4.12-4.13 versions and no longer applies as of 4.15. WDYT?

Reply:
Maybe we can remove the version numbers from this section, and only list the expected ISA? @wking WDYT?

Reply:
I'm not installer/CPU enough to be strongly opinionated here.  But I'm naively in favor of pointing out that there are constraints, and maybe there are RHEL-side docs we can link about what the minimum requirements are for the current RHEL version?  The installer/RHCOS docs folks would have to have some mechanism to bump those links when the base RHEL version changed though, instead of linking to an old KCS that is opening with discussion of etcd operator conditions.

Reply:
@wking Agree that. As a cluster administrator, I think I'd like to know the minimum requirements but a old KCS. So link a RHCOS would be better.

Reply:
FYI, I left a [comment](https://github.com/openshift/openshift-docs/pull/86171/files#r1883377806) in `modules/installation-minimum-resource-requirements.adoc`

Reply:
Thanks, all :) applied feedback (@bscott-rh and I discussed offline). Feel free to re-verify!